RackApp/lib/myproject/app.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'erb'
+require 'rack'
+require 'active_support/time'
+require 'active_support/core_ext'
+class MyController
+  class_attribute :template_name, instance_writer: false
+  self.template_name = 'template'
+  attr_reader :person, :time
+  def initialize(user, time)
+    @person = user
+    @time = time
+  end
+
+  def template(format)
+    View.new(template_name, format)
+  end
+
+  def render_template(format)
+    template(format).render.result(binding)
+  end
+end
+
+class View
+  def initialize(name, format)
+    @erb = file(name, format)
+  end
+
+  def file(name, format)
+    file_path = File.expand_path(File.join('../../..', 'templates', "#{name}.#{format}.erb"), __FILE__)
+    if File.exist?(file_path)
+      File.read(file_path)
+    else
+      'ERROR:The file doesnot exist'
+    end
+  end
+
+  def render
+    ERB.new(@erb)
+  end
+end
+
+class User
+  attr_reader :nickname
+  def initialize(nickname)
+    @nickname = nickname
+  end
+end
+
+class CountryTime
+  attr_reader :zone, :timezone
+  def initialize(zone)
+    @zone = zone
+    Time.zone = zone
+    @timezone = Time.zone.now
```

Comment:
I do not get this, Time.zone.now will return an instance of Time.

Though the name timezone is in fact equal to zone